service/db/db.go

```diff
@@ -0,0 +1,22 @@
+package db
+
+type Person struct {
```

Comment:
What's the benefit of using newly defined Person struct instead of using the thrift generated Person object?